config/webpack/webpack.bundle.config.js

```diff
@@ -7,9 +7,9 @@ module.exports = {
   entry: helpers.root('.srctmp/index.ts'),
   output: {
     path: path.resolve(__dirname, '..', '..', 'dist', 'bundles'),
-    filename: 'core.umd.js',
+    filename: 'stache.umd.js',
```

Comment:
ie, when typing `skyux release` (or something), builder would take the app's name from skyuxconfig.json?